libbeat/outputs/mode/balance.go

```diff
@@ -282,19 +279,13 @@ func (m *LoadBalancerMode) forwardEvent(
 	msg eventsMessage,
 ) bool {
 	if msg.attemptsLeft < 0 {
-		select {
-		case ch <- msg:
-			return true
-		case <-m.done: // shutdown
-			return false
-		}
+		ch <- msg
+		return true
```

Comment:
problem is, if output is unresponsive (e.g. infinite retry + network issues), the publisher will basically hang on shutdown doing no progress.